src/Dispatcher/Masa.Contrib.Dispatcher.IntegrationEvents/README.md

```diff
@@ -17,7 +18,7 @@ Install-Package Masa.Contrib.Data.EntityFrameworkCore.SqlServer // Use SqlServer
 builder.Services
     .AddIntegrationEventBus<IntegrationEventLogService>(options=>
     {
-        options.UseXXX();//Provide pub/sub capability
+        options.UseDapr();//Use Dapr to provide pub/sub capabilities, you can also choose others, remember the supplementary package and namespace
```

Comment:
`remember the supplementary package and namespace` was I remind you.

Reply:
processed